root/lib/systemd/system/titus-abmetrix@.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Titus abmetrix for container %i
+ConditionPathIsDirectory=/var/lib/titus-inits/%i/ns
+
+# If the service restarts more than 10 times in 30 seconds, let it die
+StartLimitIntervalSec=30
+StartLimitBurst=10
+
+[Service]
+EnvironmentFile=/var/lib/titus-environments/%i.env
+# Run as root (UID 0, GID 0) and with CAP_DAC_OVERRIDE so that containers with a `USER` instruction work
+ExecStart=/usr/bin/runc --root /var/run/docker/runtime-${TITUS_OCI_RUNTIME}/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE --cwd /titus/abmetrix ${TITUS_CONTAINER_ID} /titus/abmetrix/start
```

Comment:
Can you confirm this process is actually long-lived, and doesn't fork?

Reply:
yeah, it keeps running to refresh data.  Doesn't fork.